doto-frontend/src/components/pages/Settings/SettingsPage.js

```diff
@@ -100,6 +121,32 @@ const WorkingHoursPicker = props => {
                     />
                 </MuiPickersUtilsProvider>
             </div>
+            <div style={{ marginLeft: "3vw", marginTop: "3vh", textAlign: "left" }}>
+                <Button variant="outlined" color="primary" onClick={handleClickOpen}>
+                    Save
+                </Button>
+                <Dialog
+                    open={dialog}
+                    onClose={handleClose}
+                    aria-labelledby="alert-dialog-title"
+                    aria-describedby="alert-dialog-description"
+                >
+                    <DialogTitle id="alert-dialog-title">{"Want to save those changes?"}</DialogTitle>
+                    <DialogContent>
+                        <DialogContentText id="alert-dialog-description">
+                            Your time table will be re-managed automatically. Please check again.
```

Comment:
The spelling of time table should be timetable or time-table. I also think the agree or disagree should just be Ok and Cancel instead. Other than that it seems fine to me

Reply:
solved the grammar issue.

Reply:
LGTM good job!